buildSrc/src/test/java/org/elasticsearch/gradle/testclusters/TestClustersPluginIT.java

```diff
@@ -22,12 +22,10 @@
 import org.gradle.testkit.runner.BuildResult;
 import org.gradle.testkit.runner.GradleRunner;
 import org.junit.Before;
-import org.junit.Ignore;
 
 
 import java.util.Arrays;
 
-@Ignore("https://github.com/elastic/elasticsearch/issues/42453")
 public class TestClustersPluginIT extends GradleIntegrationTestCase {
```

Comment:
The usefulness of this test is pretty questionable at this point. I'd be more inclined to completely remove it.